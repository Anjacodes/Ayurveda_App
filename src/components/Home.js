import React from 'react';
import styles from './Home.module.css';

const Home = () => (
  <div className={styles.center}>
    <h2 className="text-3xl">Welcome to our page!</h2>
    <p>
      Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor
      incididunt ut labore et dolore magna aliqua. In hac habitasse platea dictumst
      quisque. A diam sollicitudin tempor id. Ut porttitor leo a diam sollicitudin
      tempor. Nullam non nisi est sit amet facilisis. Sem et tortor consequat id porta
      nibh venenatis cras sed. Ultrices tincidunt arcu non sodales neque. Ac odio
      tempor orci dapibus ultrices in iaculis nunc sed.
    </p>
  </div>
);

export default Home;
