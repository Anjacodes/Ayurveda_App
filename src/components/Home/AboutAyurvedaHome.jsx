import React from 'react';

function AboutAyurvedaHome() {
  return (
    <div className="h-[100vh] bg-[#f3efeb] text-center px-[15%] border-2 border-solid border-b-[#aa1640]">
      <h2 className="pt-[30vh] font-elanor text-6xl mb-6">Ayurveda</h2>
      <p className="mb-8 font-mundial text-lg leading-loose">The Sanskrit term Ayurveda translates to “knowledge of life,” and the principles of this ancient wisdom remind us that the entire web of life is intricately interwoven. With a unique emphasis on total wellness, the art and science of Ayurveda work to harmonize our internal and external worlds.</p>
      <a className="rounded-lg bg-[#aa1640] text-[#f3efeb] font-mundial p-2 hover:bg-[#b4aa49]" href="/ayurveda">LEARN MORE</a>
    </div>
  );
}

export default AboutAyurvedaHome;
