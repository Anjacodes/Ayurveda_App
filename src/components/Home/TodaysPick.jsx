import React from 'react';
import mandala from '../../assets/images/mandala_opacity21.png';
import product from '../../assets/images/product.png';

function TodaysPick() {
  return (
    <div className="h-[100vh] bg-[#efdcdc] relative px-[5vw] border-2 border-solid border-t-[#aa1640]" style={{ backgroundImage: `url(${mandala})`, backgroundRepeat: 'no-repeat', backgroundPosition: 'right'}} >
      <div className="flex w-full justify-between">
        <div className="flex flex-col h-[75vh] justify-between pt-[25vh]">
          <h3 className="font-mundial font-extralight">TODAY&apos;S FAVORITE PICK</h3>
          <div>
            <h2 className="text-6xl">Product Title</h2>
            <p>Product Description</p>
            <button>Call To Action</button>
          </div>
        </div>
        <div className="pt-[25vh]">
          <div className="flex justify-around w-[55vw]">
            <img src={product} alt="product" className="h-96" />
            <img src={product} alt="product" className="h-96" />
            <img src={product} alt="product" className="h-96" />
          </div>
        </div>
      </div>

    </div>
  );
}

export default TodaysPick;
