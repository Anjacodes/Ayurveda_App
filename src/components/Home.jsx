import React from 'react';
import AboutAyurvedaHome from './AboutAyurvedaHome';
import DoshasHome from './DoshasHome';
import WelcomeScreen from './WelcomeScreen';
import WhereTo from './WhereTo';

const Home = () => (
  <div>
    <WelcomeScreen />
    <AboutAyurvedaHome />
    <DoshasHome />
    <WhereTo />
  </div>
);

export default Home;
