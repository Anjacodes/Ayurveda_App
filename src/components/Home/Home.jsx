import React from 'react';
import AboutAyurvedaHome from './AboutAyurvedaHome';
import DoshasHome from './DoshasHome';
import TodaysPick from './TodaysPick';
import WelcomeScreen from './WelcomeScreen';
import WhereTo from './WhereTo';

const Home = () => (
  <div>
    <WelcomeScreen />
    <AboutAyurvedaHome />
    <DoshasHome />
    <WhereTo />
    <TodaysPick />
  </div>
);

export default Home;
