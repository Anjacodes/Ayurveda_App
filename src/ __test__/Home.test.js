import renderer from 'react-test-renderer';
import Home from '../components/Home';

it('renders Home correctly', () => {
  const tree = renderer.create(<Home />).toJSON();
  expect(tree).toMatchSnapshot();
});
