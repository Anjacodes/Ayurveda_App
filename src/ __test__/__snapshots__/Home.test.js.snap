// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Home correctly 1`] = `
<div
  className="center"
>
  <h2
    className="title"
  >
    Welcome to our page!
  </h2>
  <p>
    Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. In hac habitasse platea dictumst quisque. A diam sollicitudin tempor id. Ut porttitor leo a diam sollicitudin tempor. Nullam non nisi est sit amet facilisis. Sem et tortor consequat id porta nibh venenatis cras sed. Ultrices tincidunt arcu non sodales neque. Ac odio tempor orci dapibus ultrices in iaculis nunc sed.
  </p>
  <br />
  <p>
    Ac felis donec et odio pellentesque diam volutpat. Aenean pharetra magna ac placerat vestibulum lectus mauris ultrices. Ipsum a arcu cursus vitae congue mauris rhoncus aenean vel. Purus sit amet volutpat consequat mauris nunc congue nisi vitae. Maecenas volutpat blandit aliquam etiam erat velit scelerisque in dictum.
  </p>
</div>
`;
